files/themes/srcery.toml

```diff
@@ -0,0 +1,13 @@
+idle_bg = "#1C1B19" # Black
+idle_fg = "#FCE8C3" # Bright White
+info_bg = "#1C1B19" # Black
+info_fg = "#FCE8C3" # Bright White
+good_bg = "#1C1B19" # Black
+good_fg = "#98BC37" # Bright Green
+warning_bg = "#1C1B19" # Black
+warning_fg = "#FED06E" # Bright Yellow
+critical_bg = "#1C1B19" # Black
+critical_fg = "#F75341" # Bright Red
+separator = ""
```

Comment:
Hmm, is this intentionally empty? Maybe at least just a space would be a better default?

Reply:
Yes, this works really great for me actually. I enjoy the minimalism of it.